### ISSUE TITLE
Expired proposals (past closing time) no longer represented as such in the UI

### ISSUE THREAD

**Issue by tibetsprague** [daostack/alchemy#276]
In old Alchemy when a proposal was past it's closing time but had not yet been we moved it to the History tab with a button to Execute or Redeem rewards, clicking this button was necessary to update the proposal state on the blockchain which is why we have to handle the UX of proposal expiration entirely in Alchemy. The new code does not correctly check for expired proposals

**Comment by A-Zak** [comment]
This is what we discussed in the dev call, right?

To follow up on this, the main bug here is that a proposal that is past it's closing time is not considered "expired" as its status might not have been changed on the blockchain. @tibetsprague is that correct?

Regarding the handling of expired proposals, once a proposal expires, it should appear both in the history page and in the redemptions page for the people who staked upon it.
